src/silvimetric/resources/metrics/l_moments.py

```diff
@@ -1,32 +1,27 @@
 from ..metric import Metric
 import numpy as np
-from .p_moments import mean
-from lmoments3 import lmom_ratios
 from lmoments3 import distr
 
 import warnings
-# suppress warnings from dividing by 0, these are handled in the metric creation
-warnings.filterwarnings(
-    action='ignore',
-    category=RuntimeWarning,
-    module='lmoments3'
-)
 
 def lmom4(data):
```

Comment:
We should really provide a linkage to the names of these moments back to FUSION somehow or document the heck out of them here. In-the-know people can see the name and recognize them, but anyone else ends up having to read the code to understand what is in there.